Tasks/CargoAuthenticateV0/cargoauthenticatemain.ts

```diff
@@ -0,0 +1,57 @@
+import * as path from 'path';
+import * as tl from 'azure-pipelines-task-lib/task';
+import * as toml from 'toml';
+import * as fs from 'fs';
+import * as constants from './constants';
+
+async function main(): Promise<void> {
+    tl.setResourcePath(path.join(__dirname, 'task.json'));
+
+    try {
+        let configtoml = tl.getInput(constants.CargoAuthenticateTaskInput.WorkingFile);
+        if (!(configtoml.endsWith('.toml'))) {
```

Comment:
Cargo versions before 1.39 used `config` without the `.toml` extension, and current Cargo still supports that.

Reply:
From the [docs](https://doc.rust-lang.org/cargo/reference/config.html):

> Cargo also reads config files without the `.toml` extension, such as `.cargo/config`. Support for the `.toml` extension was added in version 1.39 and is the preferred form. If both files exist, Cargo will use the file without the extension.

Reply:
Yeah I saw that, but it wasn't very clear if the file name has to be `config` in any case even though the hierarchy scanning structure in another doc only use config.toml https://doc.rust-lang.org/stable/cargo/reference/config.html?highlight=config.toml#hierarchical-structure Should we support file that isn't name `config.toml`?

Reply:
The current behavior, it will not check for toml extension or file name, we only check if the provided file path exist.